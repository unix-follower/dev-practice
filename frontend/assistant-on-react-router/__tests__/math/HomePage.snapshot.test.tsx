import React from "react"
import "@testing-library/jest-dom"
import { render } from "@testing-library/react"
import Page from "~/math/Home"

describe("Math Home page snapshot", () => {
  test("renders home page", () => {
    const { container } = render(<Page />)
    expect(container).toMatchSnapshot()
  })
})
