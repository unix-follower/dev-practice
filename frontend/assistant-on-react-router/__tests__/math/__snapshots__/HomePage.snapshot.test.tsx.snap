// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Math Home page snapshot renders home page 1`] = `
<div>
  <span>
    Home placeholder
  </span>
</div>
`;
